gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/TTimestampWrapper.java

```diff
@@ -132,4 +156,57 @@ public int getMibiecIntValue() {
     public int getAntcfinalIntValue() {
         return timestamp.getANTCFinal().getV().intValue();
     }
+
+    //
+
+    Map<String, Boolean> getExportCornerActiveForCountryMap() {
+        if (exportingCountryMap == null) {
+            exportingCountryMap = new HashMap<>();
+            List<TCalculationDirection> calculationDirections = timestamp.getCalculationDirections().get(0).getCalculationDirection();
+            calculationDirections.forEach(tCalculationDirection -> {
+                if (tCalculationDirection.getInArea().getV().equals(eicCodesConfiguration.getItaly())) {
+                    exportingCountryMap.put(tCalculationDirection.getOutArea().getV(), false);
+                } else if (tCalculationDirection.getOutArea().getV().equals(eicCodesConfiguration.getItaly())) {
+                    exportingCountryMap.put(tCalculationDirection.getInArea().getV(), true);
+                }
+            });
+            exportingCountryMap.put(toEic(Country.IT), true);
+        }
+        return exportingCountryMap;
+    }
+
+    public boolean isExportCornerActiveForCountry(String countryEic) {
+        getExportCornerActiveForCountryMap();
+        return Optional.ofNullable(exportingCountryMap.get(countryEic))
+                .orElseThrow(() -> new CseValidInvalidDataException("Country " + countryEic + " must appear in InArea or OutArea"));
+    }
+
+    public boolean isExportCornerActiveForFrance() {
+        return isExportCornerActiveForCountry(eicCodesConfiguration.getFrance());
+    }
+
+    public Map<String, Double> getImportCornerSplittingFactors() {
+        return timestamp.getSplittingFactors().getSplittingFactor().stream()
+                .collect(Collectors.toMap(
+                    tFactor -> toEic(tFactor.getCountry().getV()),
+                    tFactor -> tFactor.getFactor().getV().doubleValue()
+                ));
+    }
+
+    public Map<String, Double> getExportCornerSplittingFactors() {
+        getExportCornerActiveForCountryMap();
+        return timestamp.getShiftingFactors().getShiftingFactor().stream()
+                .collect(Collectors.toMap(
+                    tFactor -> toEic(tFactor.getCountry().getV()),
+                    tFactor -> tFactor.getFactor().getV().doubleValue())
+                );
+    }
+
+    private String toEic(String country) {
+        return toEic(Country.valueOf(country));
+    }
+
+    private String toEic(Country country) {
+        return new EICode(country).getAreaCode();
```

Comment:
There is a risk here. Actually we build different objects that all reference EICode (splitting factors, exportingCountryMap, glsk, zonal scalable, etc) there are all like maps with keys being the eicodes of the countries. On the other hand, when we build the objects, some of the eicodes come from eiCodesConfiguration, others from PowSybl objects (Country, EICode, etc) and others from the source files directly (ttc adjustment or glsk). It is important that all these eicodes are equals otherwise we won't be able to cross information between these objects. So I would strongly advise to try to harmonize that so that all the eicodes in our final objects come from the same source. From recent decision it feels that it should come from eiCodesConfiguration, but maybe it is to discuss further with @vbochetRTE on this topic.